### PR TITLE
Claims status page connected to benefits api

### PR DESCRIPTION
**Pull request by jbalboni**
Closes: [46](https://github.com/department-of-veterans-affairs/sunsets-team/issues/46)

This is a WIP while I get some answers on the timeline events.

### CODE REVIEW

**Comment by jbalboni** [comment]
@cagedcrown This isn't done, but you can take a look at some of it when you have a chance.

**Review by molindoz — APPROVED** [review]
Nice work! Everything looks good to me 👍

**Comment by austin-martinez** [comment]
Are there tests that you can be adding as you're developing?

**Comment by jbalboni** [comment]
@austin-martinez Yeah, those are up next.

File: src/js/disability-benefits/components/ClaimPhase.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    this.showAllActivity = this.showAllActivity.bind(this);
+  }
+  displayActivity() {
+    if (this.props.activity) {
+      const filesPath = `your-claims/${this.props.id}/files`;
+      const limitList = this.state.showAll ? _.identity : _.take(INITIAL_ACTIVITY_ROWS);
+      const activityList = limitList(this.props.activity).map((activity, index) =>
+        <div key={index} className="claims-evidence">
+          <p className="claims-evidence-date">{moment(activity.date).format('MMM D, YYYY')}</p>
+          {activity.type === 'still_need_from_you_list'
```
Comment (jbalboni) [inline src/js/disability-benefits/components/ClaimPhase.jsx]: @austin-martinez Does this make sense as far as using the different event types?

Comment (austin-martinez) [inline src/js/disability-benefits/components/ClaimPhase.jsx]: Yep, looks good :+1:

File: src/js/disability-benefits/components/ClaimPhase.jsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+          {activity.type === 'still_need_from_others_list'
+            ? <p className="claims-evidence-item">We requested <Link to={filesPath}>{activity.displayName}</Link> from others</p>
+            : null}
+          {activity.type === 'received_from_others_list'
+            ? <p className="claims-evidence-item">We received {activity.displayName} from others</p>
+            : null}
+          {activity.type === 'received_from_you_list'
+            ? <p className="claims-evidence-item">We received {activity.displayName} from you</p>
+            : null}
+          {activity.type === 'phase_entered'
```
Comment (austin-martinez) [inline src/js/disability-benefits/components/ClaimPhase.jsx]: This activity type doesn't exist yet, do we need to add this? @carlmjohnson

**Pull request merged**